docs/pages/accounts/sso.mdx

```diff
@@ -59,8 +59,9 @@ Regular users may be a member of one or many personal, team, and organization ac
 To transition from using a regular Expo account to an SSO account, follow these steps:
 1. Go to the [SSO login page](https://expo.dev/sso-login) and follow the prompts, entering your organization name, creating a new Expo username, and logging in to your identity provider.
 2. By default, your new SSO user will have the View Only role. If you need a different role, ask an Admin or Owner to update your role in [member settings](https://expo.dev/accounts/[account]/settings/members).
-3. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
-4. If you no longer need your old user account, log out of your new SSO account, then login to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under your old user account.** It will not affect any projects owned by the organization.
+3. Run `eas login --sso` to switch to your new account on the CLI.
+4. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
+5. If you no longer need your old user account, log out of your new SSO account, then login to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under your old user account.** It will not affect any projects owned by the organization.
```

Comment:
```suggestion
3. Run `eas login --sso` to switch to your new account on the CLI.
4. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
5. If you no longer need your old user account, log out of your new SSO account, then log in to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under your old user account.** It will not affect any projects owned by the organization.
```

Just a small nit, "login" here should be a verb.